charts/datadog/values.yaml

```diff
@@ -2034,6 +2034,14 @@ clusterChecksRunner:
   # clusterChecksRunner.createPodDisruptionBudget -- Create the pod disruption budget to apply to the cluster checks agents
   createPodDisruptionBudget: false
 
+  containers:
```

Comment:
do you mind moving this section to be adjacent to `clusterChecksRunner.securityContext`? similar to how it's done for the [cluster agent](https://github.com/DataDog/helm-charts/blob/c71caaa562671a23c575d469569de96fb9f7260e/charts/datadog/values.yaml#L1037-L1048) . thanks